ui/e2e/src/config/base.playwright.config.ts

```diff
@@ -45,6 +45,22 @@ const config: PlaywrightTestConfig = {
     baseURL: 'http://localhost:3000',
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
     trace: 'on-first-retry',
+
+    /**
+     * Disabled per playwright documentation, so we can reliably mock network
+     * requests.
+     * https://playwright.dev/docs/network#missing-network-events-and-service-workers
+     */
+    serviceWorkers: 'block',
+
+    // Keep this in sync with the setting in `.happo.js` for consistency for
+    // canvas elements, which are converted into images when run in playwright
+    // and sent to happo.
+    viewport: { width: 1200, height: 800 },
```

Comment:
I was debating putting this out into a constant to share across the two config files, but the happo one needs to be in js, and it seemed like a small thing we'll rarely touch to keep in sync. Let me know if you'd prefer I do pull it out into a constant.